### PR TITLE
Magento_Contact: avoid using deprecated escape* methods from Abstract…

### PR DESCRIPTION
**Pull request by sergiynezbritskiy**
### Description (*)

Avoid using deprecated escape* methods from AbstractBlock

### Related Pull Requests

https://github.com/magento/magento2/pull/31610

### Contribution checklist (*)
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [x] All new or changed code is covered with unit/integration tests (if applicable)
 - [x] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Comment by ihor-sviziev** [comment]
@magento run all tests

**Comment by ihor-sviziev** [comment]
@magento run Functional Tests B2B, Functional Tests EE, Static Tests

**Comment by ihor-sviziev** [comment]
@magento run Functional Tests B2B, Functional Tests EE, Static Tests

File: app/code/Magento/Contact/view/frontend/templates/form.phtml [hunk 1]
```diff
@@ -7,79 +7,82 @@
 // phpcs:disable Magento2.Templates.ThisInTemplate
 // phpcs:disable Generic.Files.LineLength.TooLong
 
-/** @var \Magento\Contact\Block\ContactForm $block */
+/**
+ * @var \Magento\Contact\Block\ContactForm $block
+ * @var \Magento\Framework\Escaper $escaper
+ */
 /** @var \Magento\Contact\ViewModel\UserDataProvider $viewModel */
```
Comment (ihor-sviziev) [inline app/code/Magento/Contact/view/frontend/templates/form.phtml]:
```suggestion
/**
 * @var \Magento\Contact\Block\ContactForm $block
 * @var \Magento\Framework\Escaper $escaper
 * @var \Magento\Contact\ViewModel\UserDataProvider $viewModel
 */
```

**Review by ihor-sviziev — CHANGES_REQUESTED** [review]

File: app/code/Magento/Contact/view/frontend/templates/form.phtml (hunk 1 above)
Comment (sergiynezbritskiy) [reply app/code/Magento/Contact/view/frontend/templates/form.phtml]: fixed

**Comment by ihor-sviziev** [comment]
@magento run all tests

**Comment by ihor-sviziev** [comment]
@magento run Functional Tests B2B, Functional Tests CE, Functional Tests EE, Integration Tests

**Review by ihor-sviziev — APPROVED** [review]
✔ Approved.

Failing tests looks not related to changes form this PR.

**Comment by ihor-sviziev** [comment]
@magento run Functional Tests B2B, Functional Tests EE, Functional Tests CE

**Pull request closed without merging**